src/api/config.ts

```diff
@@ -19,7 +19,14 @@ class ConfigAPI {
     const deviceId = params.deviceId || Device.getDeviceId();
     const installId = params.installId || Device.getInstallId();
     const sessionId = Session.getSessionId();
-    const locationAuthorization = await Navigator.getPermissionStatus();
+
+    // location authorization
+    let locationAuthorization;
+    try {
+      locationAuthorization = await Navigator.getPermissionStatus();
+    } catch (err: any) {
+      Logger.warn(`Location authorization error: ${err.message}`);
+    }
```

Comment:
nit: i would refactor this a bit to make it less verbose esp since `getPermissionStatus` _should_ technically no longer throw

```suggestion
      const locationAuthorization = await Navigator.getPermissionStatus().catch((err: any) => {
      Logger.warn(`Location authorization error: ${err.message}`);
      }
```

but im happy to take this across the line

Reply:
I would leave as `try/catch`. Chaining the catch is generally for fire-and-forget requests. We're still using `await` here.

Reply:
sure just a code styling thing. ik we use it in a bunch of other async code paths with `await` since it removes the need for another level of nesting but not super opinionated here. just makes things a bit cleaner